Tests/SentryTests/Performance/SentryTracerTests.swift

```diff
@@ -262,13 +262,14 @@ class SentryTracerTests: XCTestCase {
         
         child.finish()
         XCTAssertEqual(2, fixture.dispatchQueue.dispatchAfterInvocations.count)
-        
+
+        // The grandchild is a NoOp span
         let grandChild = child.startChild(operation: fixture.transactionOperation)
-        XCTAssertEqual(3, fixture.dispatchQueue.dispatchCancelInvocations.count)
+        XCTAssertEqual(2, fixture.dispatchQueue.dispatchCancelInvocations.count)
         
         grandChild.finish()
-        XCTAssertEqual(3, fixture.dispatchQueue.dispatchAfterInvocations.count)
-        XCTAssertEqual(4, fixture.dispatchQueue.dispatchCancelInvocations.count)
+        XCTAssertEqual(2, fixture.dispatchQueue.dispatchAfterInvocations.count)
+        XCTAssertEqual(2, fixture.dispatchQueue.dispatchCancelInvocations.count)
```

Comment:
`h`: Hmm, why do we have to change these? I think this should still work. You should still be able to create a child span on a finished span. You only should get a no op when the transaction is finished.

Sentry can handle this see https://sentry.io/organizations/sentry-sdks/discover/sentry-cocoa:60e32f111dbb4ad18dcf3327f56f5a8d/?field=title&field=event.type&field=project&field=user.display&field=timestamp&field=device&field=os&id=15529&name=Cocoa+Events&project=5428557&query=event.type%3Atransaction&sort=-timestamp&statsPeriod=24h&topEvents=5&yAxis=count%28%29
<img width="794" alt="Screen Shot 2022-09-30 at 10 24 19" src="https://user-images.githubusercontent.com/2443292/193226404-0ec6ac16-d6a3-4479-8178-ce1905634c2a.png">

Reply:
Ok, that wasn't clear to me from the ticket. Changed.